src/app/tools/point_shapes.h

```diff
@@ -171,11 +223,11 @@ class BrushPointShape : public PointShape {
     }
 
     // TODO cache compressed images (or remove them completelly)
-    if (m_lastBrush != brush) {
+    if (m_dynamics.isDynamic())
+      setSymmetricCommpressedImages(brush);
+    else if (m_lastBrush != brush) {
       m_lastBrush = brush;
-      m_compressedImage.reset(new CompressedImage(brush->image(),
-                                                  brush->maskBitmap(),
-                                                  false));
+      setSymmetricCommpressedImages(brush);
     }
```

Comment:
Could we re-use the compressed brushes if the brush wasn't re-created? Could we just keep the `m_lastBrush != brush` for the dynamic case too (e.g. would be useful for a serie of points with similar dynamics)?